building/developing_optee_applications_in_rust.rst

```diff
@@ -0,0 +1,97 @@
+.. _developing_optee_applications_in_rust:
+
+###################################### 
+Developing OP-TEE Applications in Rust
```

Comment:
Tempted to suggest simply
`Rust Trusted Applications`
or
`Trusted Applications - Rust`

This will void wrapping the text on the menubar to the left.

Reply:
How about simplifying the title to `Teaclave TrustZone SDK`? @mssun

Reply:
> `Rust Trusted Applications`
> or
> `Trusted Applications - Rust`

Hi, I'm okay with these two. But we also provide Rust crate (i.e., `optee_teec`) to help developing host apps in Rust. I don't know how to have a simple title to express the both efforts.

Reply:
How about `OP-TEE with Rust`?

Reply:
`Trusted and Client Applications with Rust`?

Reply:
I prefer `OP-TEE with Rust`. I believe this can be rendered in the menubar without wrapping.

Reply:
Thanks all,
I'll update the title to `OP-TEE with Rust`.